hud-data/bania-hud.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "Bania Hud",
+  "": "a hud i made lolololol",
+  "author": "smh2211",
+  "contributors": [
+    "Hypnotize",
+    "Chippy",
+    "Village Green Preserver",
+    "weareallimaginary"
+  ],
+  "releaseDate": "2024-05-20",
+  "resources": [
```

Comment:
`resources` is meant to list the names of the images you'd have in `hud-resources/bania-hud`:
<img width="578" alt="chrome_KJ0vaR7y7y" src="https://github.com/mastercomfig/hud-db/assets/509599/b42b77b6-9580-405a-b8d2-8f1d24f18dff">